src/EnergyPlus/UnitarySystem.cc

```diff
@@ -13350,7 +13557,13 @@ namespace UnitarySystems {
                 }
                 this->m_ElecPower = locFanElecPower;
                 this->m_ElecPowerConsumption = this->m_ElecPower * ReportingConstant;
-
+            } else if (SELECT_CASE_var == DataHVACGlobals::CoilDX_Cooling) {
+                if (this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat) {
+                    if (CoolingLoad && this->LoadSHR == 0.0) {
+                        this->LoadSHR = 1.0;
+                        this->CoilSHR = coilCoolingDXs[this->m_CoolingCoilIndex].performance.NormalSHR;
+                    }
+                }
```

Comment:
This is in reportUnitarySystem. If there is a cooling load and LoadSHR was not calculated then set it to 1. Not sure when that would happen but this is a report variable so I guess it's OK?  Is the coil SHR > 0 and <= 1 at this time? And if so, then why wasn't a LoadSHR calculated? Is LoadSHR calculated only if performance mode is not equal normal?

Reply:
Load SHR is calculated at every mode.